mmv1/products/bigqueryreservation/CapacityCommitment.yaml

```diff
@@ -15,7 +15,7 @@
 name: 'CapacityCommitment'
 base_url: projects/{{project}}/locations/{{location}}/capacityCommitments
 create_url: projects/{{project}}/locations/{{location}}/capacityCommitments?capacityCommitmentId={{capacity_commitment_id}}
-self_link: 'projects/{{project}}/locations/{{location}}/capacityCommitments/{{capacity_commitment_id}}'
+self_link: '{{name}}'
```

Comment:
Can you also change `self_link` back?

Reply:
This change is intentional. I want `{{name}}` used in `GET` and `PATCH` because `{{capacity_commitment_id}}` may be empty.

Reply:
With this change, the get url won't work during import, as `name` is unavailable.

Reply:
Updated to use custom Import